config/read_write_database_sample.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
Do we want a sample in code? I believe having updated docs with sample in docs will have better visibility.
Also, if syntax matches what Laravel documents, then we don't really need this at all.